dir.props

```diff
@@ -276,7 +276,8 @@
     <NETCoreAppTestSharedFrameworkPath>$(NETCoreAppTestRootPath)shared/Microsoft.NETCore.App/$(NETCoreAppTestSharedFxVersion)/</NETCoreAppTestSharedFrameworkPath>
 
     <!-- Constructed shared fx path for testing -->
-    <TestSharedFxDir>$(NETCoreAppTestRootPath)</TestSharedFxDir>
+    <TestSharedFxDir Condition="'$(_bc_TargetGroup)' != 'netfx'">$(NETCoreAppTestRootPath)</TestSharedFxDir>
+    <TestSharedFxDir Condition="'$(_bc_TargetGroup)' == 'netfx'">$(RuntimePath)</TestSharedFxDir>
```

Comment:
@weshaggard this is not really needed but it was wrong anyway for NetFX. so fixing it in case we'll use it somewhere later